migrations/20210910114750_view_ibc_channel_messages.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE view_ibc_channel_messages (
+    id BIGSERIAL,
+    channel_id VARCHAR NOT NULL,
+    block_height BIGINT,
+    block_time BIGINT NOT NULL,
+    transaction_hash VARCHAR NOT NULL,
+    success BOOLEAN NOT NULL,
+    error VARCHAR NOT NULL,
+    sender VARCHAR NOT NULL,
+    receiver VARCHAR NOT NULL,
```

Comment:
Sender here sometimes mean the fungible token sender, sometimes it's the relayer. Should we consider separate a `relayer` column here instead?

Reply:
Yes, will add a `relayer` column here! Thanks!